app/controllers/avo/search_controller.rb

```diff
@@ -9,8 +11,8 @@ class SearchController < ApplicationController
 
     def show
       render json: search_resources([resource])
-    rescue => error
-      render_search_error(error)
+    rescue StandardError => e
+      render_search_error(e)
     end
```

Comment:
I don't think this is related to our PR.

Reply:
No, I changed how it was before in Refactoring PR.